web-app/src/components/common/Form/index.js

```diff
@@ -38,12 +38,10 @@ export class Form extends Component {
     for (let key of keys) {
       if (this.props.formData[key] !== '') {
```

Comment:
Looks like destructuring props around here would lead to something slightly more clean